docs/sources/tutorials/quick-start-browser.md

```diff
@@ -244,6 +273,9 @@ provider.register({
 
 const ignoreUrls = [COLLECTOR_URL];
 
+// please note that this instrumentation come from open telemetry and can't be used directly 
```

Comment:
Please be aware that this instrumentation originates from OpenTelemetry and cannot be used directly in the initializeFaro instrumentations options. If you wish to configure these instrumentations using the initializeFaro function, please utilize the instrumentations options within the TracingInstrumentation class.